src/main/java/com/conveyal/r5/streets/StreetLayer.java

```diff
@@ -1308,8 +1310,8 @@ public int createAndLinkVertex (double lat, double lon) {
      */
     public Split findSplit(double lat, double lon, double radiusMeters, StreetMode streetMode) {
         Split split = null;
-        if (radiusMeters > 300) {
-            split = Split.find(lat, lon, 150, this, streetMode);
+        if (radiusMeters > MAX_LINK_RADIUS_METERS) {
```

Comment:
I might rename `MAX_LINK_RADIUS_METERS` to `FIRST_SEARCH_RADIUS` or something like that.  If `radiusMeters` exceeds this value, a split can still be found below, which means it's not really the maximum.

The comment above could also be clarified, to require less thinking about what "The search radius can be specified freely here because we use this function to link transit stops to streets but also to link pointsets to streets, and currently we use different distances for these two things" means.

Reply:
Definitely agree, this needs to be clarified. It took me 5 minutes to even understand why I did this.